src/components/error.tsx

```
@@ -0,0 +1,55 @@
+import type { ErrorComponentProps } from "@tanstack/react-router";
+import { ErrorComponent, Link } from "@tanstack/react-router";
+
+import { House } from "@/components/icons/house";
+import { Refresh } from "@/components/icons/refresh";
+import { Button } from "@/components/ui/button";
+
+function Error({ error, reset }: ErrorComponentProps) {
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)